parachain/src/chain_spec/kintsugi.rs

```diff
@@ -257,5 +259,9 @@ fn kintsugi_mainnet_genesis(
             safe_xcm_version: Some(2),
         },
         sudo: Default::default(),
+        loans: LoansConfig {
+            max_exchange_rate: Rate::from_inner(loans::DEFAULT_MAX_EXCHANGE_RATE),
+            min_exchange_rate: Rate::from_inner(loans::DEFAULT_MIN_EXCHANGE_RATE),
```

Comment:
Yes they need to be set in the migration

Reply:
Actually they should already be set: https://github.com/interlay/interbtc/blob/6ed29e99412a4d29a5cfe461fd60db0826ba24c1/crates/loans/src/lib.rs#L439-L449

Reply:
This should happen during the upgrade when the pallet is first added, otherwise the markets won't be usable. We should double-check this with chopsticks